twisted-client-1/get-poetry.py

```diff
@@ -86,7 +86,8 @@ def connectionLost(self, reason):
         reactor.stop() # no more poetry
 
     def doRead(self):
-        bytes = ''
+        # data from socket is bytes
```

Comment:
I think we should rename the variable so that it doesn't clash with the `bytes` built-in. And no need for the comment, the `b'` prefix makes that clear.

Reply:
Renamed to chunks. Removed comment. Pushed change.